packages/frontend/scss/misc/_keyboard_hint_box.scss

```diff
@@ -47,29 +47,31 @@
 }
 
 .keyboard-hint-cheatsheet-dialog {
+  width: 80% !important;
+  max-width: 560px !important;
   .keybinding {
-    display: block;
-    flex-grow: 1;
     font-size: 0.84em;
-    margin-bottom: 0;
+  }
+  h4 .keybinding {
+    flex: none;
+    margin-left: 20px;
+  }
+  .dialog-body {
+    padding-bottom: 0;
   }
   .keyboard-hint-dialog-body {
     display: flex;
     flex-wrap: wrap;
-    flex-direction: column;
-    padding: 4px;
-    max-height: 600px;
-    justify-content: space-evenly;
-    padding: 20px;
-
+    padding-left: 10px;
+    padding-bottom: 10px;
```

Comment:
```suggestion
    justify-content: space-evenly;
    padding-left: 10px;
    padding-bottom: 10px;
```

Otherwise when there is one column it's kinda off IMO.

![image](https://github.com/user-attachments/assets/dc575475-6381-485f-88dc-70ba959aed81)

Reply:
Ok, maybe better to have a centered view.